src/components/Modal/FilterModal.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// import FooterData from '../Footer/FooterData';
+// import { moveUp, moveDown } from '../Footer/ShowEffect';
+import Filter from '../Filter/Filter';
+import { StyleFilterModal, ModalBox } from './StyleFilterModal';
+
+const FilterModal = ({ isOpenModal, handleModal }) => {
+  return (
+    <StyleFilterModal onClick={handleModal}>
+      <ModalBox isOpenModal={isOpenModal} onClick={e => e.stopPropagation()}>
+        {isOpenModal && (
+          <Filter isOpenModal={isOpenModal} handleModal={() => handleModal()} />
```

Comment:
```suggestion
          <Filter isOpenModal={isOpenModal} handleModal={handleModal} />
```
여기도 인자로 전달할 값이 없으면 위와 같이 해주세요!